modular_citadel/code/objects/items/medieval.dm

```diff
@@ -0,0 +1,139 @@
+// Helmets
+/obj/item/clothing/head/helmet/medieval
+	name = "medieval helmet"
+	desc = "A classic metal helmet, effective at stopping melee attacks."
+	icon = 'modular_citadel/icons/obj/clothing/medieval_helmet.dmi'
+	icon_override = 'modular_citadel/icons/mob/medieval_helmet.dmi'
+	icon_state = "knight"
+	item_state = "knight"
+	armor = list(melee = 41, bullet = 15, laser = 5, energy = 5, bomb = 5, bio = 2, rad = 0)
```

Comment:
why specifically 41 melee instead of 40? why specifically 2 bio instead of 0?

Reply:
Tis but a copypaste of main serb values. Going to change it ASAP.